packages/datetime/src/dateRangePicker.tsx

```diff
@@ -237,9 +237,12 @@ export class DateRangePicker extends AbstractPureComponent2<DateRangePickerProps
         return (
             <div className={classes}>
                 {this.maybeRenderShortcuts()}
-                <div>
-                    {this.renderCalendars(isShowingOneMonth)}
-                    {this.maybeRenderTimePickers()}
+                <div className={DateClasses.DATEPICKER_CONTENT}>
+                    <div>
```

Comment:
This doesn't seem like the ideal DOM structure... it probably makes more sense for the calendar(s) and time picker(s) to live adjacent to the footer element. I would like to see more options added to the docs example so we can test out this new layout. I'll try to push my ideas as a commit on your branch.

Reply:
I made an update that I think is more what you are looking for..